drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -141,7 +146,7 @@ void RasterizerSceneGLES3::shadow_atlas_set_size(RID p_atlas,int p_size){
 				       GL_TEXTURE_2D, shadow_atlas->depth, 0);
 
 		glViewport(0,0,shadow_atlas->size,shadow_atlas->size);
-		glClearDepth(0);
+		glClearDepth(1.0f);
```

Comment:
yeah I think I screwed that up when I undid those #ifdef blocks and changed it to a define up top, I'll fix it up tonight:)